### ISSUE TITLE
Minimum required fields are not intuitive

### ISSUE THREAD

**Issue by joemienko** [cssat/sprout-issues#42]

**Comment by yhtill** [comment]
Some fields are implicitly required in the input XML.  This should not be the case.

**Comment by joemienko** [comment]
@yhtill 

1. What are these fields?
2. Is this an issue we still want to resolve?
3. What is the desired behavior?

**Comment by yhtill** [comment]
1. I don't know any more -- the heart of this issue is that some of the XML parsing code inherently does not tolerate blank or missing fields in the input XML, not by design, but because the code is brittle.  I discovered this fact when we were attempting to build a canonical "minimal" XML referral as part of a manual migration process (if I recall correctly).  Basically, I started with an XML doc that just had the normal top-level element of a referral, and discovered that that failed -- then went through and figured out what needed to be added to get it to work (and did a little digging to discover roughly why).  Unfortunately, that discovery process will need to be reproduced.
2. It is a bug, in a real sense, but one that is very unlikely to be triggered in actual use of the system.  Essentially, if FamLink were for some reason to send Sprout an empty referral, Sprout would error out, instead of "degrading gracefully."  I suspect it wouldn't be a particularly large effort to fix this bug, but I don't think it's a terribly high priority.  It could be useful to fix it to enable some test scenarios.
3. The desired behavior is that Sprout doesn't have unexpected errors when parsing "empty" XML referrals, or XML referrals that are missing certain fields.
